### ISSUE TITLE
Categories

### ISSUE THREAD

**Issue by klavdiasotskova** [graphprotocol/mission-control-curator#340]
Exchanges is subcategory of DeFi. But there are exchanges which are not part of DeFi. Please add separate category.
Games should be divided into subcategories by genre.

**Comment by yanivtal** [comment]
I agree this feels wrong. Still not sure what the best category for those would be.
